modules/i18n/src/translations/en-US/portals/remote-user-stores.ts

```diff
@@ -0,0 +1,245 @@
+            connectedUserStoreType: {
+                label: "Remote User Store Type",
+                placeholder: "Select the type of remote user store",
+                validation: {
+                    required: "User store type is required"
+                }
+            },
+            description: {
+                label: "Description",
+                placeholder: "Describe the purpose or scope of this user store"
+            },
+            groupIdMapping: {
+                helperText: "Specify the attribute from the user store that stores the group ID.",
+                label: "Group ID",
+                placeholder: "Ex: groupId",
+                validation: {
+                    required: "Group ID mapping is required when read groups is enabled"
+                }
+            },
+            groupnameMapping: {
+                helperText: "Specify the attribute from the user store that stores the group name.",
+                label: "Group Name",
+                placeholder: "Ex: groupName",
+                validation: {
+                    required: "Group name mapping is required when read groups is enabled"
+                }
+            },
+            name: {
+                label: "User Store Name",
+                placeholder: "ex: MY USER STORE",
+                validation: {
+                    required: "User store name is required"
+                }
+            },
+            readGroups: {
+                helperText: "Enable this option to retrieve group information from the user store.",
+                label: "Read Groups"
+            },
+            userIdMapping: {
+                helperText: "Specify the attribute from the user store that represents the unique ID for the user.",
+                label: "User ID Mapping",
+                placeholder: "Ex: uid",
+                validation: {
+                    required: "User ID mapping is required"
+                }
+            },
+            usernameMapping: {
+                helperText: "Specify the attribute from the user store that represents the user's primary identifier.",
+                label: "Username Mapping",
+                placeholder: "Ex: un",
+                validation: {
+                    required: "Username mapping is required"
+                }
+            }
+        },
+        sections: {
+            groupAttributes: "Group Attributes",
+            userAttributes: "User Attributes"
+        }
+    },
+    notifications: {
+        connectionStatusCheckError: {
+            description: "There was an error while checking the connection status.",
+            message: "Something went wrong!"
+        },
+        disconnectError: {
+            description: "There was an error while disconnecting the user store.",
+            message: "Disconnection failed!"
+        },
+        tokenGenerateError: {
+            description: "There was an error while generating the installation token.",
```

Comment:
Do we need to show the error messages related to internal processes to the end user?